src/hooks/useInfiniteLoop.ts

```diff
@@ -0,0 +1,13 @@
+import { createElement, useEffect, useState } from "react"
+
+export function useInfiniteLoop() {
+  const [state, setState] = useState(0)
+
+  useEffect(() => {
+    setState(state + 1)
```

Comment:
Maybe add some logs here?

Reply:
Interesting 🤔
Now to either use that error message you get in dev or put a copypasta here